veps/sig-network/hotpluggable-nad-ref.md

```diff
@@ -0,0 +1,194 @@
+# VEP #140: Live Update NetworkAttachmentDefinition Reference
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [X] (R) Enhancement issue created, which links to VEP dir in [kubevirt/enhancements] (not the initial VEP PR)
+- [X] (R) Target version is explicitly mentioned and approved
+- [X] (R) Graduation criteria filled
+
+## Overview
+
+VMs using secondary networks refer to a NetworkAttachmentDefinition object's namespace and name as part of their network specification. 
+Currently, the whole network specification, including this reference - is immutable for the lifecycle of a VM.
+If a user wishes to change the network a VM is connected to (example: switching VLANs), they must modify the VM specification and restart the VM. 
+This proposal introduces the ability to update the NAD reference (networkName) of a network on a running VM through Live Migration. 
+This allows the host network connection to change without any reboot.
+
+## Motivation
+
+Currently, in order to move VMs from one network to another, the user either needs to hotplug a new interface and unplug the old one or updated the NAD reference and reboot the VM.
+This proposal would support it without reboot and without a change in the guest interfaces.
+
+## Goals
+
+- Change an existing NAD reference (spec.networks[].multus.networkName) of secondary networks using bridge binding on a running VM, and by extension the pod network plumbing.
+
+## Non Goals
+
+- Migrating between incompatible CNI types
+- Maintaining seamless network connectivity
+- Changing the network binding/plugin
+- Changing NAD reference on non-migratable VMs
+- Changing the guest's network configuration in case it have to change following the underlying network change.
```

Comment:
a bit confusing here ... can we remove words ? Does it make sense if you just say "guest network configuration updates" ? At least that's what I understood here :sweat_smile: